shell/lib/buildx.sh

```diff
@@ -41,7 +42,7 @@ if [[ $OSTYPE == "linux-gnu"* ]]; then
 
   # Take from setup-buildx Github Action
   echo "� Creating a new builder instance"
-  docker buildx create --use
+  docker buildx create --use --name "$APPNAME"
```

Comment:
This is also ran on user's machines, so perhaps we utilize a shared one instead of one per-application? Also, we probably need to `|| true` this or something smarter to prevent this from failing.

I would probably just go with `--name devbase` and share it everywhere.

Reply:
Ah yeah right, I guess cache is per repository so we don't need such unique names.